.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,63 @@
+Use the following template to create your pull request
+
+## Description
+
+Describe the changes that where made in this pull request. When possible start with a user story - short, simple descriptions of a feature told from the perspective of the person who desires the new capability. Be sure to also include the following information:
+
+1. Motivation for change
+2. What was changed
+3. How does this impact application developers
+4. Link to relevant issues and documentation
+5. Provide examples of use cases with code samples and applicable acceptance criteria
+
+Example:
+As a Blockstack developer, I would like to encrypt files using the app private key. This is needed because storing unencrypted files is unacceptable. This pull request adds the `encryptContent` function which will take a string and encrypt it using the app private key.
+
+```
+encryptContent('my data')
+
+// Running the above should result in the following encrypted data object
+{"iv":"c91...","ephemeralPK":"031...","cipherText":"d61...","mac":"e73..."}
+```
+
+For details refer to issue #123
+
+## Type of Change
+- [ ] New feature
+- [ ] Bug fix
+- [ ] API reference/documentation update
+- [ ] Other
+
+## Does this introduce a breaking change?
```

Comment:
I don't think this section makes very much sense for the blockchain come mainnet, since the blockchain has an extremely low tolerance for most kinds of breaking changes (and upgrades require a high degree of coordination with the network participants).  I think the SIP process should be used to submit potentially-breaking changes, and this PR template should warn users up-front that consensus-breaking changes and network-protocol-breaking changes will not be accepted (since this is not the appropriate venue).

Reply:
Added SIP info regarding breaking changes